Dockerfile.QA

```diff
@@ -0,0 +1,20 @@
+FROM node:10 As build
```

Comment:
What do you think about adding a comment in each dockerfile file about that all files have duplicated code and should be edited together?

Reply:
you are right. Forgot about it thx.